styles.css

```diff
@@ -0,0 +1,111 @@
+html, body{
```

Comment:
Please get Prettier in VSCode to lint CSS! https://swe101.rocketacademy.co/course-logistics/required-hardware-and-software#vscode-formatter-and-settings